The options need to be symbols not strings

mkanoor opened the pull request:
The openapi-generator uses symbols for options instead
of strings

gmcculloug commented:
@mkanoor Is there any supporting documentation for this from open-api?  I assume all this eventually ends up being json payloads, and therefore strings, so symbols seems a surprising requirement.

mkanoor commented:
@gmcculloug In their documents they always mention symbols like here.
https://github.com/mkanoor/rbac_api_client/blob/master/docs/GroupApi.md#list_groups

I just goofed up in my code and converted it to use strings instead of symbols and learnt the hardware when my program started looping for ever.
My initial implementation was using symbols, when we switched to use limit and offset that's where I made the mistake.
